powerhome/playbook
# [PLAY-607] Give Sticky Header for Dialog a z-index

#### Screens

screencap of behavioor with z-index:

https://share.getcloudapp.com/12uRXRPW

#### Breaking Changes

No

#### Runway Ticket URL

[Runway Story](https://nitro.powerhrg.com/runway/backlog_items/PLAY-607)

#### How to test this

Navigate to dialog kit page, go to 'Full Height' doc example and click on 'Medium Dialog' button to open dialog with test kits in it. Scroll to test that other kits do not scroll over sticky header. Click 'Top Center' button inside dialog to see that confirmation toast is not effected by the z-index we added to dialog.

#### Checklist:
